src/flaresolverr_service.py

```diff
@@ -287,20 +287,36 @@ def click_verify(driver: WebDriver):
     time.sleep(2)
 
 
+def get_correct_window(driver: WebDriver) -> WebDriver:
+    if len(driver.window_handles) > 1:
+        for window_handle in driver.window_handles:
+            driver.switch_to.window(window_handle)
+            current_url = driver.current_url
+            if not current_url.startswith("devtools://devtools"):
+                return driver
+    return driver
+
+
+def access_page(driver: WebDriver, url: str) -> None:
+    driver.get(url)
+    driver.start_session()  # required to bypass Cloudflare
+    driver.start_session()
```

Comment:
Please document why we need this statements twice! 🙂
I think it's only once per access_page call

Reply:
`Workaround for "challenge not detected" caused by the devtools window` - https://github.com/FlareSolverr/FlareSolverr/commit/1c855b8af0b0e4ea4d80b4950fec974fbcd02fc8